### ISSUE TITLE
WindowsError: [Error 183] Can't create a file that already exists

### ISSUE THREAD

**Issue by GianniDPC** [add-ons/service.iptv.manager#15]
This seems to happen when restarting Kodi as well as manually Refreshing the channels and guide under Channels -> Refresh channels and guide now.

```
Error Type: <type 'exceptions.WindowsError'>
Error Contents: [Error 183] Kan geen bestand maken dat al bestaat
Traceback (most recent call last):
  File "C:\Users\Gianni\AppData\Roaming\Kodi\addons\service.iptv.manager\service.py", line 8, in <module>
	service.run()
  File "C:\Users\Gianni\AppData\Roaming\Kodi\addons\service.iptv.manager\resources\lib\service.py", line 56, in run
	BackgroundService().run()
  File "C:\Users\Gianni\AppData\Roaming\Kodi\addons\service.iptv.manager\resources\lib\service.py", line 34, in run
	Addon.refresh()
  File "C:\Users\Gianni\AppData\Roaming\Kodi\addons\service.iptv.manager\resources\lib\modules\addon.py", line 83, in refresh
	IptvSimple.write_playlist(channels)
  File "C:\Users\Gianni\AppData\Roaming\Kodi\addons\service.iptv.manager\resources\lib\modules\iptvsimple.py", line 119, in write_playlist
	os.rename(playlist_path + '.tmp', playlist_path)
WindowsError: [Error 183] Kan geen bestand maken dat al bestaat
-->End of Python script error report<--
```

**Comment by GianniDPC** [comment]
See #16

**Comment by dagwieers** [comment]
Fixed by #16
